app/models/rule.rb

```diff
@@ -120,13 +123,17 @@ def cannot_be_locked_and_under_review
     errors.add(:base, 'Control cannot be under review and locked at the same time.')
   end
 
-  def error_if_locked
-    # locked = current update
-    # locked_was = before
-    # If the previous state was not locked, updates can be made.
-    return unless locked_was
+  def error_if_under_review
+    # review_requestor_id     = state after any changes
+    # review_requestor_id_was = state before any changes
+    # If the previous state was under review, then error
+    raise(RuleUnderReviewError, id) if review_requestor_id_was.present?
```

Comment:
Could this validation be `review_requestor_id.present? && review_requestor_id_was == review_requestor_id`? That would allow the saving of a record that is transitioning into the under review state, and allow the saving of a record transitioning out of the under review state but disallow all other changes.

Reply:
I can change to that logic. We'll just have the assumption that `review_requestor_id` or `locked` will never change with other attributes on the same save action